containers/monitoring-alertmanager-image/Dockerfile

```
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: MIT
+
+#     Copyright (c) 2024 SUSE LLC
+
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon.
+
+#!BuildTag: uyuni/alertmanager:%%alertmanager_patch_version%%
+#!BuildTag: uyuni/alertmanager:latest
+
+ARG BASE=bci/bci-base:15.6
+FROM $BASE
+
+MAINTAINER SUSE LLC (https://www.suse.com/)
+
+ARG PRODUCT=Uyuni
+ARG VENDOR="Uyuni project"
+ARG URL="https://www.uyuni-project.org/"
+ARG REFERENCE_PREFIX="registry.opensuse.org/uyuni"
+
+# Define labels according to https://en.opensuse.org/Building_derived_containers
+# labelprefix=org.opensuse.uyuni.alertmanager
+LABEL org.opencontainers.image.name=monitoring-alertmanager-image
```

IF the container needs to be released via MLM, this label should be named differently 
```suggestion
LABEL org.opencontainers.image.name=server-monitoring-alertmanager-image
```